README.md

```diff
@@ -110,7 +110,7 @@ Below are described some available metrics and are given examples for fetching a
 
 ### Daily Active Addresses
 
-This metric includes the number of unique addresses that participated in the transfers of given token during the day.
+This metric includes the number of unique addresses that participated in the transfers of given token during the day. For this metric you'll need to set the [api key](#configuration) and have some SAN tokens in your account.
```

Comment:
I don't think you need API key for this metric. Also not for `Token aging (burn rate)`, `TV`.... Can you check this ?

Reply:
I made a change in `sanbase` so these endpoints require the user to be logged in. This is agreed with @valo

Reply:
But here for example: https://api.santiment.net/graphiql?variables=%7B%7D&query=query%20%7B%0A%20%20transactionVolume(%0A%20%20%20%20slug:%20%22santiment%22,%0A%20%20%20%20from:%20%222018-01-01%2016:00:00Z%22,%0A%20%20%20%20to:%20%222018-06-05%2016:00:00Z%22,%0A%20%20%20%20interval:%20%2215m%22)%20%7B%0A%20%20%20%20%20%20transactionVolume,%0A%20%20%20%20%20%20datetime%0A%20%20%20%20%7D%0A%7D%0A
You can take TV without being logged in.

Reply:
But you can still get TV for: https://api.santiment.net/graphiql?variables=%7B%7D&query=query%20%7B%0A%20%20transactionVolume(%0A%20%20%20%20slug%3A%20%22etheroll%22%2C%0A%20%20%20%20from%3A%20%222018-07-01%2016%3A00%3A00Z%22%2C%0A%20%20%20%20to%3A%20%222018-07-05%2016%3A00%3A00Z%22%2C%0A%20%20%20%20interval%3A%20%2215m%22)%20%7B%0A%20%20%20%20%20%20transactionVolume%2C%0A%20%20%20%20%20%20datetime%0A%20%20%20%20%7D%0A%7D%0A
Its just in the interval [Now() - 3months,  Now() - 1day]

Reply:
Santiment slug - no timeframe restriction.
Also for DAA and Github Activity we allow historical data now, but not realtime. (that was from last week)
For TV, Burn Rate, .. - you can still get this timeframe [Now() - 3months, Now() - 1day] without apikey.